### ISSUE TITLE
Miasmv1 jitter unix API

### ISSUE THREAD

**Issue by mrphrazer** [cea-sec/miasm#170]
Hi,

some time ago someone mentioned in the IRC that the unix API functions from miasm v1  are missing. I picked out the latest commit of nux_api.py and pasted it [here](https://gist.github.com/mrphrazer/ad7eed95330c64c96782).  

Are there reasons why they are not included in miasm v2?

**Comment by p-l-** [comment]
I think the only reason is no one took the time to port these functions to the new API.

**Comment by commial** [comment]
I totally agree with @p-l- . If you have code to test them, feel free to patch `os_dep/linux_*` and submit them :smile:.
